numpyro/distributions/transforms.py

```diff
@@ -450,6 +450,34 @@ def _inverse(self, y):
     def log_abs_det_jacobian(self, x, y, intermediates=None):
         return x
 
+class SoftplusTransform(Transform):
+    r"""
+    Transform from unconstrained space to positive domain via softplus :math:`y = \log(1 + \exp(x))`.
+    The inverse is computed as :math:`y = \log(\exp(x) - 1)`.
+    """
+    def __init__(self, domain=constraints.real):
+        self.domain = domain
+
+    @property
+    def codomain(self):
+        if self.domain is constraints.real:
+            return constraints.positive
+        elif isinstance(self.domain, constraints.greater_than):
+            return constraints.greater_than(self.__call__(self.domain.lower_bound))
+        elif isinstance(self.domain, constraints.interval):
+            return constraints.interval(self.__call__(self.domain.lower_bound),
+                                        self.__call__(self.domain.upper_bound))
+        else:
+            raise NotImplementedError
+
+    def __call__(self, x):
+        return softplus(x)
+
+    def inv(self, x):
```

Comment:
you need to use `_inverse` instead of `inv` here. Since the last NumPyro release, `Transform.inv` will return an InversedTransform.